tests/data/cads-messages/sites/cds/2023/Jan/2023-01-example-of-archived-critical.md

```diff
@@ -3,7 +3,6 @@ date: 2023-01-15T11:27:13Z
 summary: A **brief description** of the message
 severity: critical
 live: false
-show_date: true
```

Comment:
It could be OK to have a test with `show_date: false` and see the falsy in the db